test/api/FhirToFsh.test.ts

```diff
@@ -111,52 +111,6 @@ describe('fhirToFsh', () => {
     expect(results.configuration).toEqual(defaultConfig);
   });
 
-  it('should load dependencies from user input', async () => {
-    // Should be able to accept both # and @ style for dependencies
-    const results = await fhirToFsh([], {
-      dependencies: ['hl7.fhir.us.core#3.1.0', 'hl7.fhir.us.mcode@1.0.0']
-    });
-    expect(results.errors).toHaveLength(0);
-    expect(results.warnings).toHaveLength(1);
-    expect(results.warnings[0].message).toMatch('Could not determine FHIR version. Using 4.0.1.');
-    expect(results.fsh).toEqual('');
-    expect(results.configuration).toEqual({
-      ...defaultConfig,
-      dependencies: [
-        { packageId: 'hl7.fhir.us.core', version: '3.1.0' },
-        { packageId: 'hl7.fhir.us.mcode', version: '1.0.0' }
-      ]
-    });
-
-    expect(loadSpy.mock.calls).toHaveLength(1);
-    expect(loadSpy.mock.calls[0][1]).toEqual([
-      'hl7.fhir.us.core@3.1.0',
-      'hl7.fhir.us.mcode@1.0.0',
-      'hl7.fhir.r4.core@4.0.1'
-    ]);
-  });
```

Comment:
I'm wondering if it might still be useful to have this test (or something similar) even though the functionality is now moved out to the `loadExternalDependencies` function. There don't seem to be any other tests that check that the `dependencies` property on the configuration option is used, and it seems like that might be good. Or maybe a simpler test to just check that `loadExternalDependencies` is called from `fhirToFsh` is sufficient?